vita3k/shader/src/spirv_recompiler.cpp

```diff
@@ -1618,7 +1701,7 @@ static std::string convert_spirv_to_glsl(const std::string &shader_name, SpirvCo
     }
     // Compile to GLSL, ready to give to GL driver.
     std::string source = glsl.compile();
-    return source;
+    return std::move(source);
```

Comment:
std::move isn't needed as we have copy elision by default since c++17.

Reply:
Ah yes, I added some std::move, realized they were not necessary and removed them. I forgot to remove this one.